### ISSUE TITLE
Unable to reproduce the reported results on ALFWorld using the released RL checkpoint

### ISSUE THREAD

**Issue by labskill-ops** [aiming-lab/SkillRL#68]
Hi, thank you for releasing your code and the RL checkpoint.

I am trying to reproduce the reported performance on the ALFWorld benchmark using the released RL model, but I have not been able to obtain the results reported in the paper.

Specifically, my evaluation consistently achieves an average success rate of around **0.84** on ALFWorld, which is noticeably different from the reported performance. (eval_in_distribution)

I would appreciate it if you could clarify the evaluation configuration. In particular:

1. What are the exact evaluation hyperparameters used in your experiments?
2. Your paper mentions a **training batch size of 64**. Is the **evaluation batch size also 64**, or is a different value used during testing?
3. Are there any additional inference settings that are required to reproduce the reported results?
4. Could you provide the exact evaluation command or a reproducible evaluation script that was used to generate the results in the paper?

Having an official evaluation script or configuration file would greatly help the community reproduce your reported numbers.

Thank you very much for your help!

**Comment by labskill-ops** [comment]
batchsize 128 --> avg 0.77
batchsize 140 --> avg 0.52

**Comment by jimmmchen** [comment]
Hi, thank you for reporting this.

  Your result of approximately 0.84 is expected. The released RL checkpoint corresponds to the “w/o Dynamic Evolution” setting, whose reported ALFWorld in-distribution success rate is 84.4%. The 89.9% result corresponds to the full SkillRL setting with dynamic skill evolution.

  The training batch size of 64 reported in the paper is the effective prompt batch (`16` prompts × `4` gradient accumulation steps). Each prompt generates `8` rollouts, so one rollout iteration contains  `16 × 8 = 128` trajectories. The evaluation batch size is configured separately as 64.

  You can run step-0 evaluation using:

  ```bash
  export MODEL_PATH=/path/to/released_rl_checkpoint

  bash examples/grpo_trainer/run_alfworld_skills.sh vllm \
      trainer.resume_mode=disable \
      trainer.val_before_train=True \
      trainer.val_only=True \
      data.val_batch_size=64 \
      env.alfworld.eval_dataset=eval_in_distribution
  ```

  Please note that changing `data.val_batch_size` also changes the ALFWorld environment workers and evaluated task set, so results using `128` or `140` are not directly comparable.

**Comment by Yukang-Lin** [comment]
Hi, could you please release the skillbank after Dynamic Evolution, that would be helpful to reproduce the final scores on paper. Thanks a lot!
